apps/microsoft-teams/frontend/src/components/config/ChannelSelectionModal/ChannelSelectionModal.tsx

```diff
@@ -22,23 +31,58 @@ interface ChannelSelectionModalProps {
 
 const ChannelSelectionModal = (props: ChannelSelectionModalProps) => {
   const { isShown, onClose, savedChannel, handleNotificationEdit, channels, error } = props;
+  const [searchQuery, setSearchQuery] = useState<string>('');
   const [selectedChannel, setSelectedChannel] = useState<TeamsChannel>(
     savedChannel ?? defaultNotification.channel
   );
-  const { title, link, emptyContent, emptyHeading, errorMessage } = channelSelection.modal;
+  const {
+    title,
+    link,
+    emptyContent,
+    emptyHeading,
+    errorMessage,
+    searchPlaceholder,
+    description,
+    button,
+  } = channelSelection.modal;
 
   useEffect(() => {
     const foundChannel = channels.find((channel) => channel.id === savedChannel.id);
     setSelectedChannel(foundChannel ?? defaultNotification.channel);
   }, [savedChannel]);
 
-  const SupplementalModalContent = ({ children }: { children: React.ReactNode }) => (
-    <ChannelSelectionSupplementalModalContent
-      onClose={onClose}
-      handleNotificationEdit={handleNotificationEdit}
-      selectedChannel={selectedChannel}>
-      {children}
-    </ChannelSelectionSupplementalModalContent>
+  /**
+   * The search input and the list of content types must be kept separate so that the rerendering
```

Comment:
looks like some copypasta from the previous PR 😁

Reply:
Yes, this is copy/pasta from contentType selection modal, but I still think it's relevant since this component is also proxying/brokering the search query from `<DebouncedSearchInput>` to `<SearchableList>` and I thought it was helpful to add a comment explaining that.  Re-reading it now, I'm not sure it would useful it is... 🤔 

And yeah, the first line mentions `content types` which is inaccurate.  So at the very least I will fix that :)